fallitem.py

```diff
@@ -17,13 +17,18 @@
 import pygame
 
 
-class Spike:
+class FallItem:
 
-    def __init__(self, x, y, flip, speed, path_length):
+    def __init__(self, x, y, flip, speed, path_length, _type = "spike"):
         self.x = x
         self.initial = y
 
+        self.type = _type
```

Comment:
This is what you should make private `self._type` as that's consistent with activities.

Reply:
I just checked where the underscore has to be for default keywords which cannot be used as variable name. Turned it the underscore should be present after the name, I will change it to `self.type_` then